django/db/models/functions/__init__.py

```diff
@@ -4,6 +4,7 @@
 from django.db.models import (
     DateTimeField, Func, IntegerField, Transform, Value,
 )
+from django.db.models.functions.date import *  # NOQA
```

Comment:
I've changed it but I'm not sure I've done the right thing. I'm importing all necessary types wrapped up in a `# NOQA`. Would you prefer I use your suggestion of moving all imports to init and current functions to base?